### PR TITLE
I changed some

### PR DESCRIPTION
**Pull request by Rage-Gaming**
I was changed progress bar to mythic progress bar , added advanced notification, changed command to a specified key ("G") and fixed progress bar working on character load time

**ON ESX NOT QB**

Complete code: https://github.com/Rage-Gaming/hh_aidoc/tree/main/hh_aidoc%5BESX%5D

### CODE REVIEW

File: hh_aidoc[ESX]/client.lua [hunk 1]
```diff
@@ -1,51 +1,51 @@
-local ESX = nil
-
-Citizen.CreateThread(function()
-	while ESX == nil do
-	   TriggerEvent('esx:getSharedObject', function(obj) ESX = obj end)
-	   Citizen.Wait(200)
-	end
-end)
-
+ESX = nil
```
Comment (BombayV) [inline hh_aidoc[ESX]/client.lua:1]: No need to make ESX global

File: hh_aidoc[ESX]/client.lua [hunk 2]
```diff
@@ -122,19 +122,43 @@ function DoctorNPC()
...
+	--Notify("Your treatment is done, you were charged: "..Config.Price)
+	Citizen.Wait(5000)
+	DeletePed(test1)
+	DeleteVehicle(test)
 	spam = true
 end
 
 
-function Notify(msg)
+function Notify(msg, state)
```
Comment (BombayV) [inline hh_aidoc[ESX]/client.lua:155]: Unused parameter of state.

File: hh_aidoc[ESX]/client.lua [hunk 3]
```diff
@@ -1,51 +1,51 @@
...
+			if IsControlJustPressed(1, 47) then
+				ESX.TriggerServerCallback('hhfw:docOnline', function(EMSOnline, hasEnoughMoney)
+					if EMSOnline <= Config.Doctor and hasEnoughMoney and spam then
+						SpawnVehicle(GetEntityCoords(PlayerPedId()))
+						TriggerServerEvent('hhfw:charge')
+						ShowAdvNotification("CHAR_PA_FEMALE", "Doctor", "", "There is no EMS in the city don't worry i will send my PA for you!")
+						--Notify("Medic is arriving")
+					else
+						if EMSOnline > Config.Doctor then
+							exports['CRP_Notify']:Alert("Doctor", "We got your signal, please wait", 5000, 'bell')
```
Comment (BombayV) [inline hh_aidoc[ESX]/client.lua:38]: Better to leave this with as a comment and uncomment the notify under this. This way there's no need to implement a dependency.

File: hh_aidoc[ESX]/client.lua [hunk 4]
```diff
@@ -122,19 +122,43 @@ function DoctorNPC()
...
+    end)
 	Wait(Config.ReviveTime)
 	ClearPedTasks(test1)
 	Citizen.Wait(500)
-	TriggerEvent('esx_ambulancejob:revive')
-	StopScreenEffect('DeathFailOut')	
-	Notify("Your treatment is done, you were charged: "..Config.Price)
-	RemovePedElegantly(test1)
-	DeleteEntity(test)
+	TriggerEvent('esx_ambulancejob:revive', GetPlayerServerId(PlayerId()))
```
Comment (BombayV) [inline hh_aidoc[ESX]/client.lua:145]:
```suggestion
	TriggerEvent('esx_ambulancejob:revive', GetPlayerServerId(PlayerId()))
```
No need for the extra argument.

Comment (Rage-Gaming) [reply hh_aidoc[ESX]/client.lua:145]: hey bro can i get your discord id, i need a help of you

Comment (BombayV) [reply hh_aidoc[ESX]/client.lua:145]: Bombay#9032

**Pull request closed without merging**